version/version_test.go

```diff
@@ -82,3 +85,44 @@ func TestAtLeast(t *testing.T) {
 		})
 	}
 }
+
+func TestGetSemantics(t *testing.T) {
+	tests := []struct {
+		fullVersion Version
+		major       string
+		minor       string
+		patch       string
+	}{
+		{
+			fullVersion: Version{"1.2.3"},
+			major:       "1",
+			minor:       "2",
+			patch:       "3",
+		},
+		{
+			fullVersion: Version{"2.2.3"},
+			major:       "2",
+			minor:       "2",
+			patch:       "3",
+		}, {
+			fullVersion: Version{"40.23.32"},
+			major:       "40",
+			minor:       "23",
+			patch:       "32",
+		},
+	}
+	for _, test := range tests {
+		t.Run("", func(t *testing.T) {
```

Comment:
```suggestion
		t.Run(test.fullVersion.GetVersion(), func(t *testing.T) {
```
![image](https://user-images.githubusercontent.com/11367982/229358827-b9348f55-3756-4124-a881-9666e4d9d16c.png)